#!/usr/bin/env python
PACKAGE = "multirobot_driver"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("robot1_xvel", double_t, 0, "Robot1 x vel",    0, 0,   1)
gen.add("robot2_xvel", double_t, 0, "Robot2 x vel",    0, 0,   1)
gen.add("robot3_xvel", double_t, 0, "Robot3 x vel",    0, 0,   1)
gen.add("robot4_xvel", double_t, 0, "Robot4 x vel",    0, 0,   1)

exit(gen.generate(PACKAGE, "multirobot_driver", "driver"))